package.json

```diff
@@ -71,9 +71,9 @@
     "once": "^1.3.0",
     "restify-clients": "1.0.0",
     "restify-errors": "3.0.0",
-    "qs": "^3.1.0",
-    "semver": "^4.3.3",
-    "spdy": "^1.26.5",
```

Comment:
Changes to qs and semver look pretty harmless, big spdy changes AFAICT from commit log. I know the tests pass, @yunong how familiar are you with the portions of the code using spdy?